core/templates/dev/head/pages/base.html

```diff
@@ -134,68 +134,69 @@
+          <div class="oppia-main-body">
+            <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation">
+            <div class="navbar-container">
+              <top-navigation-bar></top-navigation-bar>
+
+              <div class="collapse navbar-collapse oppia-navbar-collapse ng-cloak">
+                {% block navbar_breadcrumb %}
+                {% endblock navbar_breadcrumb %}
+
+                {% block local_top_nav_options %}
+                {% endblock local_top_nav_options %}
               </div>
+            </div>
+            </nav>
 
-              {{ warnings_and_loader() }}
+            <div class="oppia-top-of-page-padding">
             </div>
 
-            <noscript>
-              <div class="oppia-page-cards-container">
-                <div class="md-default-theme oppia-page-card oppia-long-text">
-                  <!-- Note to developers: We replicate the translated text inline because, without JavaScript enabled, the translation engine doesn't kick in.-->
-                  <h2>
-                    <span translate="I18N_SPLASH_JAVASCRIPT_ERROR_TITLE">We Need JavaScript in Your Browser</span>
-                    <i class="material-icons">&#xE811;</i>
-                  </h2>
-                  <p translate="I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION"
-                     translate-values="{hrefUrl: 'http://www.enable-javascript.com/'}">Oppia is a free, open-source learning platform full of interactive activities called 'explorations'.  Sadly, Oppia requires JavaScript to be enabled in your web browser in order to function properly and your web browser has JavaScript disabled.  If you need help enabling JavaScript, <a href="http://www.enable-javascript.com"/>click here.</a></p>
-                  <p translate="I18N_SPLASH_JAVASCRIPT_ERROR_THANKS">Thank you.</p>
-                </div>
+            {{ warnings_and_loader() }}
+          </div>
+
+          <noscript>
+            <div class="oppia-page-cards-container">
+              <div class="md-default-theme oppia-page-card oppia-long-text">
+                <!-- Note to developers: We replicate the translated text inline because, without JavaScript enabled, the translation engine doesn't kick in.-->
+                <h2>
+                  <span translate="I18N_SPLASH_JAVASCRIPT_ERROR_TITLE">We Need JavaScript in Your Browser</span>
+                  <i class="material-icons">&#xE811;</i>
+                </h2>
+                <p translate="I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION"
+                translate-values="{hrefUrl: 'http://www.enable-javascript.com/'}">Oppia is a free, open-source learning platform full of interactive activities called 'explorations'.  Sadly, Oppia requires JavaScript to be enabled in your web browser in order to function properly and your web browser has JavaScript disabled.  If you need help enabling JavaScript, <a href="http://www.enable-javascript.com"/>click here.</a></p>
```

Comment:
Indent so that left edge of translate-values matches left edge of translate on previous line.

(Generally, follow previous style.)